integration-tests/general/threadsafe-enclave/build.gradle

```diff
@@ -11,7 +11,7 @@ conclave {
     productID = 1
     revocationLevel = 0
     runtime = runtimeType
-    maxThreads = 10         // To match assumptions in tests
+    maxThreads = 40         // To match assumptions in tests
```

Comment:
We now default to 100, so perhaps we can just delete this?

Reply:
This was added by @ac101m yesterday, with the intention to be explicit I think (as this enclave is also used to test the max number of TCs). So I would still make this intention clear somehow and not rely on the default value of the SDK?

Reply:
Fair enough.

Reply:
Marco is correct, some of the tests were assuming the default value. Best to be explicit.